Common/UnitDefinitions/SpecificVolume.json

```diff
@@ -21,15 +21,15 @@
       ]
     },
     {
-      "SingularName": "CubicFeetPerPound",
-      "PluralName": "CubicFeetsPerPounds",
-      "FromUnitToBaseFunc": "x/0.062427959968012",
-      "FromBaseToUnitFunc": "x*0.062427959968012",
+      "SingularName": "CubicFootPerPound",
+      "PluralName": "CubicFeetPerPounds",
```

Comment:
@mp9007 Kindly rename PluralName to `CubicFeetPerPound`. :D We keep the `PerPound` singular.
Aside from that, everything looks good +1

Reply:
@mp9007 Just a reminder that we are waiting for you to make this change. Also run `generate-code.bat` once more afterwards.